package.json

```
@@ -31,7 +31,8 @@
         "format": "yarn d2-style apply",
         "lint": "yarn d2-style check",
         "e2e": "yarn playwright test --ui",
-        "generate-types": "rm -rf ./src/types/generated && yarn openapi --input https://dev.api.im.dhis2.org/swagger.yaml --output ./src/types/generated --client axios --useOptions --useUnionTypes --exportCore false --exportServices false --exportModels true --exportSchemas false && yarn prettier src/types/generated --write --loglevel silent"
+        "generate-types": "rm -rf ./src/types/generated && yarn openapi --input https://dev.api.im.dhis2.org/swagger.yaml --output ./src/types/generated --client axios --useOptions --useUnionTypes --exportCore false --exportServices false --exportModels true --exportSchemas false && yarn prettier src/types/generated --write --loglevel silent",
+        "prepare": "husky install"
```

I know that this is the official recommendation, but I think it's easier to just do:

```suggestion
        "postinstall": "husky install"
```

This ensures it's always working, even when checking out the repo for the first time, without having to do these steps manually (and therefore having to know that you have to do these steps)